frontend/test/resolution/testOperatorOverloads.cpp

```diff
@@ -256,7 +256,7 @@ static void test5() {
       record R {
         var x : int;
       }
-      operator R.==(a: R, b: R) { return "true"; }
+      operator R.==(a: R, b: R) { return 4.2; }
```

Comment:
Here as well. Is this a workaround?

Reply:
see comment above -- it's just avoiding loading the stdlib, which is not relevant to the test at hand.